response_operations_ui/views/respondents.py

```diff
@@ -302,24 +302,32 @@ def edit_contact_details(respondent_id):
     )
 
 
-@respondent_bp.route("/resend_verification/<respondent_id>", methods=["GET"])
+@respondent_bp.route("/resend-verification/<respondent_id>", methods=["GET"])
 @login_required
 def view_resend_verification(respondent_id):
     logger.info("Re-send verification email requested", respondent_id=respondent_id)
     respondent = party_controller.get_respondent_by_party_id(respondent_id)
-    email = respondent["pendingEmailAddress"] if "pendingEmailAddress" in respondent else respondent["emailAddress"]
+    is_new_email_verification_request = True if "pendingEmailAddress" in respondent else False
```

Comment:
This might not work as you're expecting it.  `<key> in <dict>` will return True if the key is present, even if the value is false.  Or am I misunderstanding it, and a pendingEmailAddress key will only be there if present, and it's absent if there isn't one?

Reply:
I believe pendingEmailAddress key will only be present if there is one. I just tested it locally and it seems to work.